app/src/main/java/cz/martykan/forecastie/utils/UnitConvertor.java

```diff
@@ -112,6 +112,23 @@ else if (wind < 32.6) {
         }
     }
 
+    public static String convertUvIndex(double value) {
+        /* based on: https://en.wikipedia.org/wiki/Ultraviolet_index */
+        if (value < 0) {
```

Comment:
Not sure why there's a pipe character in the string. It's not shown when there's an error. If it's part of the formatting in the UI, you should make this pipe character part of the UI code. This would be helpful when others are changing the UI again. They wouldn't have to figure out why this pipe character magically appears with the UV-index.

Reply:
The pipe is only a gosmetic thing. It separates the string from the number (the actual uv index value) like "Low | 0.3". That means the risk is low and the current UV Index value is 0.3.

Wondering if I should keep the risk text only. What do you guys go for? Any other idea of how to show the values to the user?

Reply:
Based on programming best-practices, I'd recommend the following.

This method should return only the risk level as a string (e.g. "Low", Medium", etc.). You may want to change the name to `convertUvIndexToRiskLevel()` to more clearly indicate this. You would then create this string with the pipe character in `MainActivity` where you use this conversion method. That would then be:
```
double uvIndex = todayWeather.getUvIndex();
String uvIndexAndRiskLevel = String.format("%s | %s", convertUvIndexToRiskLevel(uvIndex), uvIndex);
```
This should make the code easier to understand for others and clearly shows what is being displayed in the GUI.

As I already mentioned, I'm not a maintainer of this project. So there are just tips based on Java best-practices and personal experience. If the maintainers want something else, probably best to stick with that.